cudamapper/samples/sample_cudamapper.cpp

```diff
@@ -73,7 +73,8 @@ void process_batch(std::vector<IndexDescriptor>& query_index_descriptors,
                    const std::shared_ptr<const io::FastaParser> query_parser,
                    const std::shared_ptr<const io::FastaParser> target_parser,
                    DefaultDeviceAllocator allocator,
-                   bool print)
+                   bool print,
+                   OutputFormat format)
```

Comment:
`const` to prevent accidental edits and possibly make it easier for compiler to optimize?

Reply:
good idea, fixed.